src/py/flwr/server/superlink/linkstate/utils.py

```diff
@@ -213,3 +195,141 @@ def has_valid_sub_status(status: RunStatus) -> bool:
     if status.status == Status.FINISHED:
         return status.sub_status in VALID_RUN_SUB_STATUSES
     return status.sub_status == ""
+
+
+def make_taskins_unavailable_taskres(taskins_id: Union[str, UUID]) -> TaskRes:
```

Comment:
can we use a better name here? like `create_taskres_unavailable_for_taskins` ?

Reply:
Plus, this method and the one below don't have consistent documentation (for inputs and returns) like the rest of the methods.